### PR TITLE
[WIP] Update new navs

### PR DESCRIPTION
**Pull request by beneliflo**
Don't Merge Yet

**Desktop View:**
desktop_preview

### CODE REVIEW

File: src/components/DropdownList/DropdownList.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+              />
+            : null
+          }
+        </div>
+      : null
+    }
+    {data.subItems
+      ? <div className={cx('multi-list')}>
+          {data.subItems.map(subItem =>
+            <ul role="menubar" aria-hidden="false" key={subItem.titleList}>
```
Comment (ramasilveyra) [inline src/components/DropdownList/DropdownList.js]: It's fine that this attr `aria-hidden` it's always `false`? Maybe we can remove it.

File: src/components/DropdownList/DropdownList.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                  closeDropdowns={closeDropdowns}
+                />
+              )}
+            </ul>
+          )}
+        </div>
+      : null
+    }
+    {data.items
+      ? <ul role="menubar" aria-hidden="false">
```
Comment (ramasilveyra) [inline src/components/DropdownList/DropdownList.js]: Same comment as above:
> It's fine that this attr `aria-hidden` it's always `false`? Maybe we can remove it.

Comment (beneliflo) [inline src/components/DropdownList/DropdownList.js]: We can

**Comment by beneliflo** [comment]
Thanks @ramasilveyra for that magic! :D

Mobile still WIP but all feedback is good to the final release ;)

**Comment by landitus** [comment]
Just a few observations: 

screen shot 2017-05-09 at 5 27 17 pm 1
- Solutions: Ratio de 60/40 but with a narrow dropdown width 

screen shot 2017-05-09 at 5 27 08 pm
- Docs: Right side aligns with global container.

screen shot 2017-05-09 at 5 33 29 pm
- All: Check section paddings to match design

screen shot 2017-05-09 at 5 35 34 pm
- All: align vertically with the caret halfway through the padding

- All: All icons are in color. The hover effect darkens the text only.
- All: Use new Fakt typeface
- New Talk to sales menu item with border (not a button class)

**Comment by landitus** [comment]
Looks great! Just a few comments:

- The titles of the sections (Modern Identity Platform, Use Cases, Solutions, etc) should be medium weight (500)
- The titles of the Platform items (Lock, Passwordless, etc) should also be medium weight (500)
- The titles of the subsections (INDUSTRIES, GET STARTED, SECTIONS) should also be medium weight (500)
- The b2b,b2c,b2e pills have the old font Avenir. Also, the text is not vertically centered to the pill. And the pill is not quite centered vertically with the label of the menu item.
- In Documentation, the tech icons are not quite centered vertically with the labels.
- Can we try with 30px for the left/right paddings for all modules? I think that's it :)

**Comment by landitus** [comment]
We should check the Wordpress icon SVG export in Platform. I think we need to convert it to outlines, because the export is not 100% on par with the design

**Comment by landitus** [comment]
Looking great! 
- [ ] We should check the Wordpress icon SVG export in Platform. I think we need to convert it to outlines, because the export is not 100% on par with the design
- [ ] Mobile: there's a scrolling bug in iOS, where the scroll is not working-
- [ ] Mobile: Lets add a margin right of 5px to the hamburguer icon
- [ ] Mobile: Check the weights of the titles (Modern Identity Platform, Solutions, Pricing) should be Medium (500)
- [ ] Mobile: All buttons should be Medium too
- [ ] Mobile: check the styles of the login button, it needs the same letter spacing and font-weight as the others
- [ ] Mobile: Check the scroll overflow at the top where there's a line that we should delete.
img_6062
- [ ] Mobile:  Check the the bottom, where there background grey in not covering the rest of the screen.  
img_6063

**Comment by landitus** [comment]
@beneliflo can we add 18px of padding-top only to the header elements of each dropdown?
screen shot 2017-05-16 at 6 00 31 pm


It compensates the header a bit better

**Comment by ramasilveyra** [comment]
I started to release beta versions of this under the `4.x` branch https://github.com/auth0/web-header/tree/4.x.

You can take a look to the staging app of the website with the new web-header.

Lets start to send little prs to `4.x` branch fixing the bugs that we found.

**Pull request closed without merging**